### PR TITLE
add aarch64 android complile method to doc

### PR DESCRIPTION
**Pull request by dzyong**
The install doc doesn't intro how to compile 64-bit Caffe2 for android.

@Maratyszcza please help me to review this.
Thanks.

### CODE REVIEW

**Comment by Maratyszcza** [comment]
It would be better to mention ARMv7 vs ARM64, as Android supports other 32-bit and 64-bit architectures as well

File: _docs/getting-started/android.md [hunk 1]
```diff
@@ -22,11 +22,20 @@ git clone --recursive https://github.com/caffe2/caffe2.git
 
 ## Run the Build Script
 
+If you want to build 32-bit Caffe2:
```
Comment (Maratyszcza) [inline _docs/getting-started/android.md]: Please change to "If you want to build Caffe2 for Android with armeabi-v7a ABI"

File: _docs/getting-started/android.md [hunk 2]
```diff
@@ -22,11 +22,20 @@ git clone --recursive https://github.com/caffe2/caffe2.git
...
+
 ```
 cd caffe2
 ./scripts/build_android.sh
 ```
 
+Or if you want to build 64-bit Caffe2:
```
Comment (Maratyszcza) [inline _docs/getting-started/android.md]: Please change to "Or if you want to build Caffe2 for Android with arm64-v8a ABI"

**Comment by dzyong** [comment]
Hi @Maratyszcza, thanks for your help : )
I've changed it as you say.

**Comment by Maratyszcza** [comment]
Thank you for your contribution!

**Pull request merged**